geschaeftsordnung-fachschaftsrat.md

```diff
@@ -111,7 +111,7 @@ Bei der Anwesenheit von Mitgliedern des Fachschaftsrates während seiner Sitzung
 
 (2) Die Sitzungsprotokolle werden allen Mitgliedern des Fachschaftsrates unverzüglich nach der Sitzung zugänglich gemacht.
 
```

Comment:
```suggestion
(2) Die Sitzungsprotokolle werden allen Mitgliedern des Fachschaftsrates spätestens einen Werktag nach der Sitzung zugänglich gemacht.

```

Reply:
```suggestion
(2) Die Sitzungsprotokolle werden allen Mitgliedern des Fachschaftsrates innerhalb von einem Arbeitstag nach der Sitzung zugänglich gemacht.
```

Reply:
Würde die Arbeitstagdefinition gerne in eigenen Absatz verlagert haben, da ihr Scope ja für mehr Absätze gilt als nur für diesen.

Reply:
@BenBals Wie kann man das am schönsten machen? Einfach in einen neuen Absatz machen? Ist ja eigentlich keine eigene Bestimmung, nur ein Makro